api/src/main/java/org/pmiops/workbench/db/dao/BillingProjectBufferEntryDao.java

```diff
@@ -8,12 +8,22 @@
 @Repository
 public interface BillingProjectBufferEntryDao  extends CrudRepository<BillingProjectBufferEntry, Long> {
 
+  String ASSIGNING_LOCK = "ASSIGNING_LOCK";
```

Comment:
nit: private?

Reply:
it looks like `private` modifier is not supported in interfaces for Java8